examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/PPORun.py

```diff
@@ -0,0 +1,154 @@
+            "sgd_minibatch_size": args.sgd_minibatch_size,
+            "horizon": 200,
+            "train_batch_size": args.train_batch_size,
+            "num_workers": args.num_workers,
+            "num_gpus": args.num_gpus,
+            "batch_mode": args.batch_mode,
+            "observation_filter": args.observation_filter
+        }
+        # Stop criteria for training
+        stop = {
+            "timesteps_total": 100e6,
+        }
+        if train:
+            # Init the wandb providing the project name
+            # wandb.init(project=PROJECT_NAME)
+            # Train the model using Ray tune
+            tune.run(ppo.PPOTrainer,
+                     config=model_config,
+                     stop=stop,
+                     callbacks=[WandbLoggerCallback(
+                                project=projectName,
+                                api_key_file="../../../wandb.txt",
+                                log_config=True)],
+                     local_dir=output,
+                     checkpoint_freq=wandbSaveFreq,
+                     checkpoint_at_end=True
+                     )
+            ray.shutdown()
+        else:
+            # Crate an agent having the predifined model hyperparameters
+            agent = ppo.PPOTrainer(config=model_config)
+            # Restore the train weights
+            agent.restore(
+                "./results/Deepbots-khr3hv-Ray-reward-weight,prev_pos,obs-x,y,z,vel-done-0.5-RNN-5,clip_range-0.2,lr-0.0001_6778500_steps")
+
+            # Use the enviroment of khr3hv Robot
+            khr3hv_env = khr3hvRobotSupervisor(
+                projectName, wandbSaveFreq, train, useRay)
+            # Reset the enviroment and test the trained agent
+            obs = khr3hv_env.reset()
+            khr3hv_env.episodeScore = 0
+            while True:
+                action, _states = agent.compute_action(obs)
+                obs, reward, done, _ = khr3hv_env.step(action)
+                khr3hv_env.episodeScore += reward  # Accumulate episode reward
+
+                if done:
+                    print("Reward accumulated =", khr3hv_env.episodeScore)
+                    khr3hv_env.episodeScore = 0
+                    obs = khr3hv_env.reset()
+    else:
+        # Use of the Stable-Baseline framework
+
+        # Use the enviroment of khr3hv Robot
+        khr3hv_env = khr3hvRobotSupervisor(
+            projectName, wandbSaveFreq, train, useRay)
+
+        if train:
+            # Init the wandb providing the project name
+            wandb.init(project=projectName)
+
+            # Save a checkpoint every 1000 steps
+            checkpoint_callback = CheckpointCallback(save_freq=wandbSaveFreq, save_path=output,
+                                                     name_prefix=projectName)
+
+            # Change the neural net size or the activation function
+            policy_kwargs = dict(  # activation_fn=th.nn.ReLU,
+                net_arch=[dict(pi=[64, 64], vf=[64, 64])])
```

Comment:
Made the choices more clear. th.nn.ReLU is wrong, what's `th`?
```suggestion
            # Set the neural network size and/or the activation function
            # policy_kwargs = dict(activation_fn=th.nn.ReLU, dict(pi=[64, 64], vf=[64, 64])])
            policy_kwargs = dict(dict(pi=[64, 64], vf=[64, 64])])
```

Reply:
Fixed with
```python
            # Set the neural network size and/or the activation function
            # policy_kwargs = dict(activation_fn=th.nn.ReLU, dict(pi=[64, 64], vf=[64, 64])])
            policy_kwargs = dict(net_arch=[dict(pi=[64, 64], vf=[64, 64])])
```